sdk-lib/src/main/java/cash/z/ecc/android/sdk/block/CompactBlockProcessor.kt

```diff
@@ -325,13 +328,14 @@ class CompactBlockProcessor internal constructor(
                     } else {
                         null
                     },
-                    lastDownloadRange = if (initialInfo.lastDownloadedHeight != null && initialInfo.lastScannedHeight != null && initialInfo.networkBlockHeight != null) {
+                    lastDownloadRange = if (initialInfo.networkBlockHeight != null) {
```

Comment:
This and the next few lines is the fix for block downloads.

As I review this, I think there might be one more issue I need to resolve.